Documentation/Books/Manual/Analyzers/README.md

```diff
@@ -0,0 +1,208 @@
+the underlying *type* and the query filtering and sorting functions that the
+result can be used with. For example the *text* type will produce
+*frequency* + *norm* + *position* and the `PHRASE(...)` function requires
+*frequency* + *position* to be available.
+
+Currently the following *features* are supported:<br/>
+- *frequency*: how often a term is seen, required for PHRASE(...)
+- *norm*: the field normalization factor
+- *position*: sequentially increasing term position, required for PHRASE(...)
+              if present then the *frequency* feature is also required
+
+## Analyzer usage
+
+For Analyzer usage in the context of ArangoSearch Views please see the section
+[ArangoSearch Views](../Views/ArangoSearch/README.md).
+
+The value transformation capabilities of a given analyzer can be invoked via
+the `TOKENS(...)` function to for example:
+- break up a string of words into individual words, while also optionally
+  filtering out stopwords, applying case conversion and extracting word stems
+- parse CSV/TSV or other delimiter encoded string values into individual fields
+
+The signature of the `TOKENS(...)` function is:<br/>
+`TOKENS(<value-to-parse>, <analyzer-name-to-apply>)`<br/>
+will currently accept any string value, and an analyzer name, and will produce
+an array of zero or more tokens generated by the specified analyzer
+transformation.
+
+## Analyzer management
+
+The following operations are exposed via JavaScript and REST APIs for analyzer
+management:<br/>
+- *create*: creation of a new analyzer definition
+- *get*: retrieve an existing analyzer definition
+- *list*: retrieve a listing of all available analyzer definitions
+- *remove*: remove an analyzer definition
+
+### JavaScript
+
+The JavaScript API is accessible via the `@arangodb/analyzers` endpoint from
+both server-side and client-side code, e.g.
+```
+var analyzers = require("@arangodb/analyzers");
+```
+
+The *create* operation is accessible via:
+```
+analyzers.save(<name>, <type>[, <properties>[, <features>]])
+```
+Where *properties* can be represented either as a string, an object or a null
+value and *features* is an array of string encoded feature names.
+
+The *get* operation is accessible via:
+```
+analyzers.analyzer(<name>)
+```
+
+The *list* operation is accessible via:
+```
+analyzers.toArray()
+```
+
+The *remove* operation is accessible via:
+```
+analyzers.remove(<name> [, <force>])
+```
+
+Additionally individual analyzer instances expose getter accessors for the
+aforementioned definition attributes:
+```
+analyzer.name()
+analyzer.type()
+analyzer.properties()
+analyzer.features()
+```
+
+### REST
+
+The REST API is accessible via the `/_api/analyzer` endpoint URL callable via
+HTTP requests.
+
+The *create* operation is accessible via the *POST* method on the URL:
+```
+/_api/analyzer
+```
+With the Analyzer configuration passed via the body as an object with
+attributes: <br/>
```

Comment:
shouldn't you just redirect to the relevant section in the http book ?

Reply:
How would you do that, aren't those sections autogenerated, i.e. how would you know the name of the file beforehand?
Additionally the explanations here are more verbose and explain both the JavaScript and REST APIs

Reply:
You have to link to the `.html`-file.